src/App.js

```diff
@@ -1,28 +1,25 @@
-import logo from './logo.svg';
 import './App.css';
-import { Header } from 'semantic-ui-react';
+import '@fontsource/roboto';
+import React from 'react';
+import Button from '@material-ui/core/Button';
 
-function App() {
-  return (
-    <div className="App">
-      <Header as='h1'>Welcome to the GA4GH Starter Kit</Header>
-      <Header as='h2'>This app is built with Semantic UI React</Header>
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  render(){
+    return (
+      <div className="App">
+        <meta
+          name="viewport"
+          content="minimum-scale=1, initial-scale=1, width=device-width"
+        />
+        <h1>Welcome to the GA4GH Starter Kit</h1>
```

Comment:
Let's change these header elements to Material UI `<Typography>` components to ensure consistent typing throughout the project

Reply:
see the [Typography API](https://material-ui.com/api/typography/)